fadecandy_driver/src/ros/fadecandy_driver_ros.h

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright (c) 2021 Eurotec, Netherlands
+ * All rights reserved.
+ *
+ * Author: Jad Haj Mustafa
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions
+ * are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright
+ * notice, this list of conditions and the following disclaimer.
+ * 2. Redistributions in binary form must reproduce the above
+ * copyright notice, this list of conditions and the following
+ * disclaimer in the documentation and/or other materials provided
+ * with the distribution.
+ * 3. Neither the name of the copyright holder nor the names of its
+ * contributors may be used to endorse or promote products derived
+ * from this software without specific prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS
+ * FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE
+ * COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+ * INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+ * BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+ * LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+ * LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+ * ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ * POSSIBILITY OF SUCH DAMAGE.
+ */
+
+#include <diagnostic_updater/diagnostic_updater.h>
+#include <fadecandy_msgs/LEDArray.h>
+#include <ros/publisher.h>
+#include <ros/subscriber.h>
+
+#include "../fadecandy_driver.h"
+
+namespace fadecandy_driver
+{
+class FadecandyDriverROS
+{
+public:
+  //!
+  //! \brief FadecandyDriverRos fadecandy driver ROS wrapper
+  //! \param restart_patience Restart patience
+  //!
+  FadecandyDriverROS(double restart_patience);
+
+  //!
+  //! \brief run Listen to LED messages and publishes diagnostic of the driver
+  //!
+  void run();
+
+private:
+  //!
+  //! \brief driver_ Fadecandy driver
+  //!
+  FadecandyDriver driver_;
```

Comment:
Nitpick: I think it will be great, if we can follow [Google style guide](https://google.github.io/styleguide/cppguide.html#Declaration_Order) for class member and member function declaration.

Reply:
I think in general for ROS development, we should follow http://wiki.ros.org/CppStyleGuide#Member_variables

Reply:
Yes. That is correct, we follow ROS cpp guidelines as well. Nothing biggie. I was just curious about the declaration order.